components/radio/style/index.tsx

```diff
@@ -479,6 +479,80 @@ const getRadioButtonStyle: GenerateStyle<RadioToken> = (token) => {
   };
 };
 
+/**
+ * Accessibility in high contrast, Because of the use of non-semantic tags, the only way to do this without changing the dom element is `forced-colors: active`
+ * issue: https://github.com/ant-design/ant-design/issues/41821
+ */
+const genHighContrastStyle: GenerateStyle<RadioToken> = (token) => {
+  const { componentCls } = token;
+  const radioInnerPrefixCls = `${componentCls}-inner`;
+  const baseWrapperPrefixCls = `${componentCls}-wrapper`;
+  const buttonWrapperPrefixCls = `${componentCls}-button-wrapper`;
+
+  return {
+    '@media screen and (forced-colors: active)': {
+      [`${baseWrapperPrefixCls}`]: {
+        [`${componentCls}-disabled ${radioInnerPrefixCls}`]: {
+          borderColor: 'GrayText',
+          backgroundColor: 'GrayText',
+        },
+
+        [`${componentCls}-checked ${radioInnerPrefixCls}`]: {
+          borderColor: 'Highlight',
+          backgroundColor: 'Highlight',
+        },
+      },
+
+      // Grouping `soild` and  `radio button` together
+      [`&, ${componentCls}-group-solid`]: {
+        [`${buttonWrapperPrefixCls}`]: {
+          [`&:not(${componentCls}-button-wrapper-disabled):hover`]: {
+            color: 'Highlight',
+          },
+
+          '&:not(:first-child)::before': {
+            backgroundColor: 'initial',
+            transition: 'none',
+          },
+
+          [`&-checked:not(${componentCls}-button-wrapper-disabled)`]: {
+            color: 'Highlight',
+            borderColor: 'Highlight',
```

Comment:
im wondering, isnt there a way to put the string "Highlight" into an enum, otherfiles it feels like error prone..

Reply:
I don't know the solution yet, need help from core members. 😭